.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_API_BASE_SERIES_URL='https://api.themoviedb.org/3/tv'
```

Comment:
You forgot the file in the .gitignore.

Reply:
Hi Christoph, thank you very much!
The error was that the naming of the file was wrong with the upper and lower case.

I added the `.env` into `.gitignore`